docroot/themes/contrib/civictheme/includes/listing.inc

```diff
@@ -63,22 +69,15 @@ function _civictheme_listing_element(Paragraph $paragraph) {
   \Drupal::service('theme.manager')->alter('civictheme_listing_view_name', $view_name);
 
   $view_display = 'civictheme_listing_block';
-  $show_filters = (int) $paragraph->field_c_p_show_filters->value;
 
   $view = Views::getView($view_name);
   if ($view) {
     $view->setDisplay($view_display);
+
     _civictheme_listing_alter_view($paragraph, $view);
-  }
+    $view->civictheme_listing_entity = $paragraph;
```

Comment:
Attach the listing paragraph to the view so the paragraph can be preprocessed by the view. The reason why I have moved this to the view is because views get preprocessed on every filtering request ie if using ajax or on page load and so we can preprocess the view for every request not just page load.